pkg/analyze/analyze.go

```diff
@@ -214,6 +215,51 @@ func (a *Analyzer) RunLoop(iter fileiter.Iterator) error {
 	return nil
 }
 
+func (a *Analyzer) RunLoopWithMultipleIterators(iters []fileiter.Iterator) error {
+	var wg sync.WaitGroup
+	linesChan := make(chan []byte, 8192)
```

Comment:
Might not need a buffer this large, some 16 or `2 * len(iters)` should suffice. Note that the unit is not "bytes", but \# of objects.